airflow/www/views.py

```diff
@@ -2187,9 +2188,33 @@ def get_int_arg(value, default=0):
             paging=wwwutils.generate_pages(current_page, num_of_pages,
                                            search=arg_search_query,
                                            showPaused=not hide_paused),
+            now=dt.datetime.now,
+            airflow_timezone=settings.TIMEZONE,
+            client_timezone=CLIENT_TIMEZONE,
+            stringify_timedelta=stringify_timedelta,
             auto_complete_data=auto_complete_data)
 
 
+def stringify_timedelta(delta: dt.timedelta) -> str:
+    seconds = delta.total_seconds()
+    sign_word = ' ago' if seconds < 0 else ' from now'
```

Comment:
If you do this:
```suggestion
    sign_word = ' ago' if seconds < 0 elif seconds > 0 ' from now' else 'now'
```

Reply:
python doesn't allow you to extend a ternary in that way.

Reply:
I am pretty sure this is valid Python:
```suggestion
    sign_word = ' ago' if seconds < 0 else (' from now' if seconds > 0  else 'now')
````

Reply:
Actually even shorter if you use the `signum` of `seconds`
```python
sign_word = ['ago', 'now', 'from now'][1 + signum(seconds)]
```

Then in your last line, you can simply do:
```python
return ' '.join(words + sign_word)
```